airbyte-integrations/connectors/source-jdbc/src/main/resources/jdbc_models/jdbc_models.yaml

```diff
@@ -10,10 +10,18 @@ properties:
 definitions:
   JdbcState:
     type: object
-    additionalProperties: false
     required:
       - streams
```

Comment:
Is streams still required if it's one or the other?

Reply:
good point. seems like it shouldn't be.

Reply:
fixed.